src/components/data-processing/CellSizeDistribution/CalculationConfig.jsx

```diff
@@ -17,6 +17,8 @@ const CellSizeDistributionConfig = (props) => {
     config, disabled, updateSettings, highestUmi,
   } = props;
 
+  const withinRange = (cellSize) => Math.max(Math.min(cellSize, highestUmi ?? 17000), 0);
```

Comment:
Thanks! I was not aware of the `??` operator!
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Its very useful to avoid ifs!